pandas/tests/series/test_apply_relabeling.py

```diff
@@ -0,0 +1,33 @@
+import pandas as pd
+import pandas._testing as tm
+
+
+class TestNamedAggregation:
+    def test_relabel_no_duplicated_method(self):
+        # this is to test there is no duplicated method used in agg
+        df = pd.DataFrame({"A": [1, 2, 1, 2], "B": [1, 2, 3, 4]})
+
+        result = df["A"].agg(foo="sum")
+        expected = df["A"].agg({"foo": "sum"})
+        tm.assert_series_equal(result, expected)
+
+        result = df.B.agg(foo="min", bar="max")
```

Comment:
Can you use label access instead of dot notation? Applicable in a few places

Reply:
yeah, changed for all in this file